common-lib/src/main/scala/com/gu/mediaservice/lib/argo/model/EmbeddedEntity.scala

```diff
@@ -19,7 +20,8 @@ object EmbeddedEntity extends WriteHelpers {
   implicit def embeddedEntityWrites[T: Writes]: Writes[EmbeddedEntity[T]] = (
     (__ \ "uri").write[String].contramap((_: URI).toString) ~
       (__ \ "data").writeNullable[T] ~
-      (__ \ "links").writeNullable[List[Link]].contramap(someListOrNone[Link])
+      (__ \ "links").writeNullable[List[Link]].contramap(someListOrNone[Link]) ~
+      (__ \ "actions").writeNullable[List[Action]].contramap(someListOrNone[Action])
```

Comment:
Yes the idea is to abstract this in theseus, something like:

``` js
resource.perform('action').
  then(resp => /* result of performing the request */).
  catch(err => {
    // no such action error, or request failed error, or callback failed error
  })
```

The same way as we do:

``` js
resource.follow('link').
  then(res => /* resource after following link */`).
  catch(err => {
    // no such rel error, or request to `resource` error, or callback failed error
  })
```

Links are omitted if none and I think it makes sense to do the same for actions. It makes the response lighter and also more backward compatible.